src/sdk-reference/java/1/collection/search-specifications/snippets/search-specifications.test.yml

```diff
@@ -0,0 +1,9 @@
+name: collection#searchSpecifications
+description:
+hooks:
+  before:
+  after:
+template: default
+expected: Success
```

Comment:
Shouldn't this be `Successfully retrieved 1 specifications`?

Reply:
:+1: